setup.py

```diff
@@ -22,6 +22,13 @@
     ]
 
 
+extras_require = {
+    'tests': tests_require,
+    'deletepermission': [
+        'collective.deletepermission',
```

Comment:
Ok then, I'd like to leave it as it is, because that's the way we usually do that.
See https://github.com/4teamwork/ftw.simplelayout/blob/master/setup.py for reference.